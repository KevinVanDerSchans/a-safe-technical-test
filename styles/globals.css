@tailwind base;
@tailwind components;
@tailwind utilities;

:root {
  --light-accent: #0284c7;
  --dark-accent: #4f46e5;
  --solarized-accent: #e07a5f;
}

body,
html {
  height: 100%;
  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
  background-color: var(--background);
  color: var(--foreground);
  scroll-behavior: smooth;
}

*::-webkit-scrollbar {
  width: 0.5rem;
}

*::-webkit-scrollbar-track {
  background: linear-gradient(#000, #808080, #000);
}

*::-webkit-scrollbar-thumb {
  height: 5rem;
  background-color: var(--accent);
}

@layer utilities {
  .text-balance {
    text-wrap: balance;
  }
}
