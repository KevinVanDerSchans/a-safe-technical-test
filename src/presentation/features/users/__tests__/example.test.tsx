import { render } from '@testing-library/react'
import Example from './example'
import { expect } from '@jest/globals'

it('renders Example unchanged', () => {
  const { container } = render(<Example />)

  expect(container).toMatchSnapshot()
})
