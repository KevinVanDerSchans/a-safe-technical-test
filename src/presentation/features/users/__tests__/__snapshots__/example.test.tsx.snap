// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Example unchanged 1`] = `
<div>
  <h1>
    Example
  </h1>
</div>
`;
